doc/sources/installation/installation-linux-venvs.rst

```diff
@@ -123,14 +123,14 @@ Replace `python` with `python3` for Python 3.
     # load the virtualenv
     source ~/kivy_venv/bin/activate
 
-    # if you'd like to be able to use the x11 winodw backend do:
+    # if you'd like to be able to use the x11 window backend do:
     export USE_X11=1
 
     # install the correct Cython version
     pip install |cython_install|
 
     # Install stable version of Kivy into the virtualenv
-    pip install --no-binary kivy
```

Comment:
`--no-binary` *is* needed. This is instructions for how to install from source. Without `--no-binary` it'll install the wheel, which we're trying to avoid here because we want to compile it ourselves. 

For some systems the apparently doesn't seem to work, so they have to do this. The fix should be to list `kivy` as the `--no-binary` argument. I think I made the same mistake elsewhere in the docs (of missing the `kivy` argument).

Reply:
Ha. Yes, it's still wrong though. It's missing the fot.
```
$ pip install --no-binary kivy
You must give at least one requirement to install (see "pip help install")
```
If should be ```$ pip install --no-binary kivy .```, I think? Change made accordingly.

Reply:
It shouldn't be a dot because they haven't cloned so it's not local and they are getting it from pypi. So it should be `pip install kivy --no-binary kivy`

Reply:
That works. Done.